### ISSUE TITLE
Configure Multiple Schedulers

### ISSUE THREAD

**Issue by dharmangmakwana-panamax** [kubernetes/website#43156]
gcr.io/my-gcp-project/my-kube-scheduler:1.0

While Creating the custom scheduler, this image is not available for the general public. If my understanding is correct then my-gcp-project is private and not publically available to eveyone.

**Comment by pranav-pandey0804** [comment]
hi @dharmangmakwana-panamax ,
We sincerely appreciate your inquiry and your effort in seeking assistance. While we value your question, we would like to suggest that the GitHub issue tracker may not be the most ideal platform for obtaining advice on Kubernetes-related matters. We kindly recommend exploring alternative resources, such as the [Kubernetes Slack community](http://slack.k8s.io/) or the Discuss [Kubernetes forum](https://discuss.kubernetes.io/), where you can often find the comprehensive support you are seeking. Thank you for your understanding.

**Comment by dharmangmakwana-panamax** [comment]
Well noted Mr pranav

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pranav-pandey0804** [comment]
@dharmangmakwana-panamax for reference, check out this: https://github.com/kubernetes/kubernetes/blob/master/SUPPORT.md

**Comment by pranav-pandey0804** [comment]
@dharmangmakwana-panamax Furthermore,  the issue arises from the fact that the custom Kubernetes scheduler image is hosted in a private Google Container Registry, and access to this image is restricted. 
You can adjust the permissions for the image or make it public if that is acceptable in your use case.

**Comment by pranav-pandey0804** [comment]
@dharmangmakwana-panamax you can close this issue by commenting /close

**Comment by niranjandarshann** [comment]
> gcr.io/my-gcp-project/my-kube-scheduler:1.0
> 
> While Creating the custom scheduler, this image is not available for the general public. If my understanding is correct then my-gcp-project is private and not publically available to eveyone.

yes, if the image of container used in custom scheduler is private it can  only be restricted for authorized user . It will not be available to everyone.

**Comment by niranjandarshann** [comment]
So here it seems that my-gcp-project is private which will not be available to everyone.

**Comment by sftim** [comment]
`gcr.io/my-gcp-project/my-kube-scheduler:1.0` is an _example_ and not an actual project we expect people to use.

The page could be clear about this to help people to whom it is not obvious.
/kind bug
/remove-kind support

/retitle Configure Multiple Schedulers content confuses some readers
